MAINTENANCE.md

```diff
@@ -0,0 +1,294 @@
+    - Go to [Release notes](https://forum.scylladb.com/c/scylladb-release-notes/18) section.
+    - Click "New Topic".
+    - Title should be `[RELEASE] ScyllaDB Rust Driver <version>`, e.g. `[RELEASE] ScyllaDB Rust Driver 0.14.0`
+    - Tags: `release`, `drivers`, `rust-driver`, `rust`.
+    - Content of the post should just be release notes.
+    - Click "Create Topic"
+    - Posts in "Release notes" section need additional confirmation. You can write to current forum admin to expedite this.
+
+You're done!
+
+
+## Writing release notes
+
+PR titles are written for maintainers, but release notes entries are written for users.
+It means that they should not be the same - and if they are then they are probably
+not good as either PR titles or release notes entries.
+
+For that reason we hand-write our release notes, and link to relevant PRs in the entries.
+
+Some old release notes that you can take inspiration from when writing new ones:
+- https://github.com/scylladb/scylla-rust-driver/releases/tag/v0.10.0
+- https://github.com/scylladb/scylla-rust-driver/releases/tag/v0.14.0
+- https://github.com/scylladb/scylla-rust-driver/releases/tag/v0.11.0
+- https://github.com/scylladb/scylla-rust-driver/releases/tag/v0.13.0
+
+The template for release notes can be found in the section below.
+
+Guidelines on how to write release notes:
+
+- Go over all the PRs / commits since previous release. Usually: `git log --first-parent` to see
+  merge commits and commits that are directly on a branch. You can also try filtering
+  merged PRs on Github by merge date, but it's cumbersome.
+
+- Items in release notes will usually correspond 1:1 to PRs / comits - but not always. It is possible that
+  some functionality that should be a single item on the list is split over multiple PRs.
+  It is also possible that single PR will be mentioned in two items.
+
+- Release notes for a version should only contain items that make it different
+  from previous version. Example: commit `A` that is on main was backported to
+  `branch-0.13.x`, and `v0.13.2` was released with it - then release `0.14` will
+  not include `A` in release notes, but `0.13.2` will.
+
+- Release notes for **major** version should contain a table with number of commits per contributor.
+  You can generate this data with a command 
+  `git shortlog $(git merge-base main previous_version_tag)..HEAD -s -n`
+  (or `git shortlog previous_major_version_tag..HEAD -s -n`, assuming that a major
+  version tag is on a commit present in `main` branch, as it should be).
+  This table should not count version bump commits - subtract them from your
+  row if you already created them.
+
+- Release notes for **minor** version should NOT contain the aforementioned list.
+
+- This creates a bit of a weird situation: if a patch was backported, then it will
+  be present in release notes for some (minor) version(s), but in a commit table
+  for another (major) version. We accept this weirdness and don't fight it. 
```

Comment:
Embrace the weirdness!

Reply:
No more weirdness after I changed the procedure according to your comment below - so this note will be removed